recipe/meta.yaml

```diff
@@ -26,18 +26,19 @@ requirements:
     - python
     - click >=6.6
     - cloudpickle >=0.2.2
-    - cytoolz >=0.7.4
+    - cytoolz >=0.8.2
     - dask-core >=2.9.0
     - msgpack-python >=0.6.0
     - psutil >=5.0
     - pyyaml
     - sortedcontainers !=2.0.0,!=2.0.1
     - tblib >=1.6.0
-    - toolz >=0.7.4
+    - toolz >=0.8.2
     - tornado >=5  # [py<38]
     - tornado >=6.0.3  # [py>=38]
     - zict >=0.1.3
     - setuptools
+    - openssl !=1.1.1e
```

Comment:
This really should be in the `run_constrained` section, https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained.  Distributed does to depend on openssl directly, rather if openssl is installed in the environment the version should not be 1.1.1e.  

I would not argue for a new build with the change since there is little harm in having it as a run requirement but it would be good to move this for the next release.  The worst is that openssl will be pulled into an environment where it is not needed, although I believe openssl is a requirement for all `python` packages so it should already be required.

Reply:
Sounds good, I've opened up #124 as a reminder for when we do the next release. Thanks for following up here @jjhelmus